.travis/all-java.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<ruleset name="All Java Rules" 
+    xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+    <description>Every Java Rule in PMD</description>
```

Comment:
We can add in the description, that this is used for the regression tests with pmdtester